datafusion/core/src/physical_planner.rs

```diff
@@ -223,7 +223,7 @@ fn create_physical_name(e: &Expr, is_first_expr: bool) -> Result<String> {
             create_function_physical_name(&fun.to_string(), false, args, Some(order_by))
         }
         Expr::AggregateFunction(AggregateFunction {
-            func_def,
+            func: func_def,
```

Comment:
nit is that we could rename the local variable to func as well for more consistency